README.md

```diff
@@ -1,54 +1,136 @@
-# Terraform Enterprise: Clustering for Google
+# Terraform Enterprise Clustered on Google Cloud Platform
 
-![Terraform Logo](https://github.com/hashicorp/terraform-google-terraform-enterprise/blob/master/assets/TerraformLogo.png?raw=true)
+The **terraform-enterprise** Terraform module is the official way for
+HashiCorp customers to provision [Terraform Enterprise Clustered][tfe]
+on Google Cloud Platform (GCP).
 
-## Description
+![architecture diagram][architecture-diagram]
```

Comment:
This isn't a blocker for the review but this is a VERY high level diagram, and we may want to update these both in the repos as well as in the docs https://github.com/hashicorp/terraform-google-terraform-enterprise/pull/55/files#diff-04c6e90faac2675aa89e2176d2eec7d8R12 to be more precise.